.travis.yml

```diff
@@ -1,9 +1,17 @@
 language: node_js
 node_js:
-- '5'
+  - '5'
 before_install:
-- npm install
+  - npm install
 # before-script:
-# - grunt test cover
+#   - grunt test cover
 script:
-- npm run travis
+  - npm run travis
+env:
```

Comment:
Happy to merge this one in if we can remove the environmental changes, or at least explain why they're useful. Is this a local configuration?